iOS/Flows/Conversation/Cells/ConversationMessageCell.swift

```diff
@@ -19,7 +19,7 @@ class ConversationMessageCell: UICollectionViewCell {
     var handleDeleteMessage: ((Messageable) -> Void)?
 
     private let collectionView: UICollectionView = {
-        let layout = UICollectionViewFlowLayout()
+        let layout = ConversationMessageCellLayout()
```

Comment:
Using a custom layout because we need custom attributes to change the cell colors among other things.